pyEpiabm/pyEpiabm/tests/test_func/helper_func.py

```diff
@@ -50,25 +51,24 @@ def compare_susceptible_groups(self, small_pop, large_pop,
 
         """
         for age_group in range(len(pe.Parameters.instance().age_proportions)):
-            with self.subTest(age_group=age_group):
-                if method == 'greater':
-                    self.assertNonZeroGreater(
+            if method == 'greater':
+                self.assertNonZeroGreater(
```

Comment:
Are we summing over age groups here? I would prefer to test each age group separately if possible, but if we can't do that then we certainly don't need to iterate over them in L53! 

The gold standard here would be to that each age group category is greater or equal (i.e. no age group is worse off under the intervention), and also check the sum is strictly greater (i.e. there is a net positive impact of the intervention). I can help write this if you have difficulties, but it should be fairly straightforward from replicating content here, and will just need a detailed docstring to explain what you are doing.